<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,50">
   <property name="leftMargin">
    <number>100</number>
   </property>
   <property name="topMargin">
    <number>200</number>
   </property>
   <property name="rightMargin">
    <number>100</number>
   </property>
   <item row="0" column="1">
    <widget class="QPushButton" name="download_btn">
     <property name="minimumSize">
      <size>
       <width>493</width>
       <height>114</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #F23026;
border-radius: 25px;

font-family: 'Gilroy';
font-style: normal;
font-weight: 500;
font-size: 31px;
line-height: 38px;
text-align: center;

color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>скачать файл</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="back_btn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>65</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border-image: url(:/newPrefix/off.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="logo_r.qrc"/>
 </resources>
 <connections/>
</ui>
